### ISSUE TITLE
Make upgrading of Wasp easier

### ISSUE THREAD

**Issue by Martinsos** [wasp-lang/wasp#111]
Right now, Wasp is installed via installer script. There is no special support for upgrading Wasp, you just run installer script and it installs the latest version.

If something is already installed on the locations where Wasp wants to install, with the same name as Wasp, or if there is previous installation of Wasp, Wasp will warn user to delete whatever is there to continue.
Those warning are currently coming one after another, meaning that installation fails, user deletes stuff, and then it fails again with another "delete that" message, and user has to delete stuff again -> it would be nicer if instead he was told at once: these locations needs to be deleted. It can also be confusing because locations have simlar paths so he could think it is same message again which means he did smth wrong but he didn't.

Further, we should probably look into upgrading being a part of the Wasp itself, done with smth like `wasp upgrade`.

TLDR: Instead of reporting "delete this location because there is smth already there" errors one by one, installer script should report all of them (2 for now) at once.

**Comment by Martinsos** [comment]
Done (reporting all errors at once).
